src/main/java/ch/njol/skript/expressions/ExprTargetedBlock.java

```diff
@@ -77,14 +88,8 @@ public Class<Block> getReturnType() {
 	}
 
 	@Override
-	public boolean setTime(int time) {
-		super.setTime(time);
-		return true;
-	}
```

Comment:
I agree with removing the time, but this makes this pull request a breaking change.
That's weird having a time state on this.

Reply:
this one got me.
in what scenario would there ever be a time for this.
How/what would it break exactly?

Reply:
If someone used `target block will be {_block}` it will be `cannot understand` anymore.

Reply:
ahh ok, ive never seen anyone use that.
Ill put it back